src/common/inc/blobs.h

```diff
@@ -18,50 +18,34 @@
 #include <stdint.h>
 #include "blob.h"
 #include "pixytypes.h"
-#include "colorlut.h"
 #include "qqueue.h"
 
-#define MAX_BLOBS             100
+#define MAX_BLOBS             20
 #define MAX_BLOBS_PER_MODEL   20
 #define MAX_MERGE_DIST        7
-#define MIN_AREA              20
+#define MIN_AREA              10
```

Comment:
what's the intent of decreasing the min area?

Reply:
Just seeing how well this will work with increasing detection range. We can change back if we see issues.